examples/add_points.py

```diff
@@ -30,9 +30,11 @@
     layer.visible = False
     layer.visible = True
 
-    # change the layer selection
-    layer.selected = False
-    layer.selected = True
+    # select the layer
```

Comment:
@jni this is one of the most consequential end user changes in this PR

Reply:
Yes I love it! :blush: